.github/workflows/e2e.yaml

```diff
@@ -27,6 +27,9 @@ on:
       create-namespace:
         type: boolean
         default: false
+      append-java-version:
```

Comment:
maybe this should be an optional `flink-java-version` and the matrix could contain the versions we want to test